bcipy/task/paradigm/rsvp/copy_phrase.py

```diff
@@ -94,7 +95,8 @@ class RSVPCopyPhraseTask(Task):
         'font', 'fixation_color', 'trigger_type',
         'filter_high', 'filter_low', 'filter_order', 'notch_filter_frequency', 'down_sampling_rate', 'prestim_length',
         'is_txt_stim', 'lm_backspace_prob', 'backspace_always_shown',
-        'decision_threshold', 'max_inq_len', 'max_inq_per_series', 'max_minutes', 'max_selections', 'min_inq_len',
+        'decision_threshold', 'max_inq_len', 'max_inq_per_series', 'max_minutes', 'max_selections', 'max_incorrect',
```

Comment:
As a reminder to myself, we need to add this parameter to MatrixCopyPhrase in the orchestrator PR. I have since separated them.